@ndlib/gatsby-theme-marble/src/components/Internal/Seo/index.js

```diff
@@ -32,20 +32,22 @@ export const Seo = ({
       }
     `
   )
-  const frontmatter = typy(data, 'remarkMarblePage.frontmatter').safeObject
   const { siteMetadata } = site
+  const itemTitle = title || typy(data, 'marbleItem.title').safeString
+  const itemDescription = description || typy(data, 'marbleItem.description').safeString
+  const itemAuthor = author || typy(data, 'marbleItem.metadata[0].value[0]').safeString
```

Comment:
This may not always be an author/creator: `typy(data, 'marbleItem.metadata[0].value[0]').safeString`. 

Make sure to do a `typy` check on `marbleItem.metadata` before you do this:
```
const  creator = marbleItem.metadata.find(md => {
  return md.label === 'Creator'
}
```
Then use `typy(creator, 'value[0]').safeString. Feel free to cram this into a function to make it cleaner to follow and unit test.

Reply:
Updated to:
```
const creator = typy(data, 'marbleItem.metadata').safeArray.find(md => {
    return md.label === 'Creator'
  })
const itemAuthor = author || typy(creator, 'value[0]').safeString
```